lib/framework.js

```diff
@@ -27,23 +26,17 @@ function setupCoverageReporting(fullConfig) {
   fullConfig.coverageReporter.browserId = 'name';
 }
 
-// modified from example at:
-// https://stackoverflow.com/questions/8188548/splitting-a-js-array-into-n-arrays
-function splitArray(flatArray, numCols){
-  const maxColLength = Math.ceil(flatArray.length/numCols);
-  const nestedArray = _.chunk(flatArray, maxColLength);
-  for (var i = nestedArray.length; i < numCols; i++) {
-    nestedArray.push([]);
-  }
-  return nestedArray;
+function setupSets(config, basePath, files) {
+  config.sets = config.getSets(config, basePath, files);
+  var lastSet = config.sets && config.sets.length && config.sets[config.sets.length - 1];
```

Comment:
If we use lodash, why not simplify this expression as well? `_.last(config.sets)`

Reply:
Good idea